includes/functions/functions_prices.php

```diff
@@ -1549,7 +1549,7 @@ function zen_update_salemaker_product_prices($salemaker_id)
 {
     global $db;
     $zv_categories = $db->Execute("SELECT sale_categories_selected FROM " . TABLE_SALEMAKER_SALES . " WHERE sale_id = " . (int)$salemaker_id);
-    if ($zv_categories->EOF) return false;
+    if ($zv_categories->EOF || empty($zv_categories->fields['sale_categories_selected'])) return false;
```

Comment:
Please split the `if` clause from the action taken, i.e. no single line if/action statements.